ngx-build

```diff
@@ -314,7 +314,13 @@ sub apply_patches {
     chdir ".." or die "cannot switch to ..\n";
 
     chdir "nginx-$version" or die "cannot switch to nginx-$version\n";
-    shell("patch -p1 < $root/../openresty/patches/nginx-$version-ssl_pending_session.patch");
+    if ($ver ge '001015008') {
+        shell("patch -p1 < $root/../openresty/patches/nginx-$version-ssl_pending_session.patch");
```

Comment:
This branch should be applying `ssl_sess_cb_yield.patch`, and not `ssl_pending_session.patch`.

Reply:
Pushed the fix to the PR